test/Microsoft.AspNetCore.Mvc.DataAnnotations.Test/Internal/DataAnnotationsMetadataProviderTest.cs

```diff
@@ -589,6 +601,49 @@ public void CreateDisplayMetadata_IsFlagsEnum_ReflectsModelType(Type type, bool
             Assert.Equal(expectedDictionary, context.DisplayMetadata.EnumNamesAndValues);
         }
 
+        [Fact]
+        public void CreateDisplayMetadata_DisplayName_Localized()
```

Comment:
Suggest `CreateDisplayMetadata_DisplayName_LocalizeWithStringLocalizer()` to clearly differentiate from tests using `ResourceType` approach.